source/sdk/lang/equalities.ooc

```diff
@@ -0,0 +1,47 @@
+// The following functions were moved from HashMap.ooc
```

Comment:
I chose not to since it isn't a class. This is the case in `source/net` for instance (although we didn't write that of course). Would you prefer a capital `E` anyways?